src/scripts/Build-Toolkit.ps1

```diff
@@ -118,3 +118,28 @@ Get-ChildItem "$PSScriptRoot/../templates/$Template*" -Directory -ErrorAction Si
 
     Write-Host ''
 }
+
+# Package optimization engine
```

Comment:
nit: Should we move this into a separate Build-OptimizationEngine script? 🤔 I'm fine keeping it here. Just thinking out loud. Fwiw, the templates should probably be moved to a Build-Template script as well. Alternatively, we can keep it here for now and define a more general approach to solve this consistently. I'm fine with anything. Just wanted to share that to see what your thoughts are.

Reply:
Well, if we move templates and AOE to a different script, why don't we move workbooks and Bicep registry modules as well? Maybe each toolkit component type should have its own build script following some naming pattern and then we had the Build-Toolkit script calling all of them, provided the component directory (templates, workbooks, aoe, etc.) had a build script... But I'd keep it as is for now. I am adding a TODO to review the logic. If you prefer, I can open an issue for this.